keyvault/2016-10-01/swagger/keyvault.json

```diff
@@ -1060,7 +1083,8 @@
           "Certificates"
         ],
         "operationId": "SetCertificateContacts",
-        "description": "Sets the certificate contacts for the specified key vault.",
+        "summary": "Sets the certificate contacts for the specified key vault.",
+        "description": " ",
```

Comment:
I've gone through the entire file and filled in placeholders, per the note below. Working with @pomortaz to get fuller descriptions as the responsibility for this is with the dev team.